assets/training/model_evaluation/environment/context/conda.yaml

```diff
@@ -25,4 +25,8 @@ dependencies:
   - statsmodels==0.11.0
   - lightgbm==3.2.1
   - msrest==0.6.21
-  - mltable=={{latest-pypi-version}}
\ No newline at end of file
+  - mltable=={{latest-pypi-version}}
+  - cryptography==41.0.2
+  - scipy==1.10.0
+  - aiohttp===3.8.5
+  - certifi===2023.07.22
```

Comment:
why are we adding this explicitly? this was never a dependency in ME components. @skasturi .. Is this a new requirement from azureml-assets?

Reply:
Done. Reverted model evaluation.